src/pybind/mgr/rook/module.py

```diff
@@ -265,26 +265,28 @@ def describe_service(self, service_type=None, service_name=None,
         num_nodes = len(self.rook_cluster.get_node_names())
 
         spec = {}
-        spec['mon'] = orchestrator.ServiceDescription(
-            spec=ServiceSpec(
-                'mon',
-                placement=PlacementSpec(
-                    count=cl['spec'].get('mon', {}).get('count', 1),
-                ),
-            ),
-            size=cl['spec'].get('mon', {}).get('count', 1),
-            container_image_name=image_name,
-            last_refresh=now,
-        )
-        spec['mgr'] = orchestrator.ServiceDescription(
-            spec=ServiceSpec(
-                'mgr',
-                placement=PlacementSpec.from_string('count:1'),
-            ),
-            size=1,
-            container_image_name=image_name,
-            last_refresh=now,
-        )
+        if service_type == 'mon' or service_type is None:
```

Comment:
When will `service_type` be set, and when will it not be set? Or what issue is this fixing?

Reply:
```orch ls [<service_type>] [<service_name>] [--export] [plain|json|json-pretty|yaml] [--refresh] ```

Either all services can be listed or based on `service_type` or `service_name`. This patch adds support for listing based on `service_type`.